### DISCUSSION TITLE
请问查询面板/词典面板从选中项（也就是激活项）背景色、前景色的css参数名称是什么？

### DISCUSSION THREAD

**Discussion by tyf2018** [xiaoyifang/goldendict-ng#1489] [Q&A] [answered]
如图所示，请问查询面板/词典面板从选中项（也就是激活项）背景色、前景色的css参数名称是什么？找了很久没找到，F12审查元素查不到那里

程序版本：版本 24.02.16-alpha.20240409.aac1e059

![image](https://github.com/xiaoyifang/goldendict-ng/assets/56342788/308d5e3f-6639-432a-935c-aab57e64e444)

**Comment by shenlebantongying** [comment]
This one:

https://github.com/xiaoyifang/goldendict-ng/blob/c062ab232d213c89077b3c170dcf2cf16f2ce376/src/stylesheets/qt-lingoes-blue.css#L6-L14

The selected item is controlled by `selection-color` and `selection-background-color`.

The side panels are `QListWidget`s which you can find extra documentation here https://doc.qt.io/qt-6/stylesheet-reference.html

**Reply by tyf2018** [reply]
@shenlebantongying 万分谢谢，问题已解决。不过想再请教一下滚动条的相关css参数名称是什么？如下图所示

![image](https://github.com/xiaoyifang/goldendict-ng/assets/56342788/02d94099-8620-456f-bda3-65568bc04aa7)

**Reply by shenlebantongying** [reply]
`QScrollBar`

Note that it has multiple components. There are related examples in the link above.

**Comment by tyf2018** [comment]
根据指引，我在`qt-style.css`中添加了下述代码，但是只在`设置面板`中的滚动条生效，而不是`主窗口`中的。请问我应该怎样修改CSS代码？
```
/*垂直滚动条*/
QScrollBar:vertical {
	background-color: rgb(50, 53, 57) !important;
}
QScrollBar::handle:vertical {
	background-color: rgb(233, 186, 133) !important;
}
```

**Reply by shenlebantongying** [reply, marked as answer]
The one beside the article view belongs to the browser's realm (very recent version of chrome).

You can modify them by adding scroll bar related CSS to `article-style.css`. 

Try searching "Chrome modify scroll bar CSS" or something like that.

https://developer.mozilla.org/docs/Web/CSS/::-webkit-scrollbar

![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/80f88de6-04f1-405c-a975-62758317bee2)

**Comment by tyf2018** [comment]
非常感谢，问题已解决！！
